icu4j/maven-build/README.md

```diff
@@ -118,3 +118,29 @@ where `<projectlist>` is a comma-separated list of names of the subfolders which
 If you want to run only a specific test(s), use the `-Dtest="<test>"` option, where `<test>` can be a test name, a class name / package prefix, or a comma-separate list of them.
 
 If you want to skip tests, use the `-DskipTests=true` option.
+
+#### Special Note about the Resources
+Currently, the Maven build reads the resources NOT from `icu4j/main/shared/data/icudata.jar` but from the individual files checked into [icu4j/maven-build/maven-icu4j-datafiles/src/main/resources/com/ibm/icu/impl/data/icudt73b](https://github.com/unicode-org/icu/tree/main/icu4j/maven-build/maven-icu4j-datafiles/src/main/resources/com/ibm/icu/impl/data/icudt73b) directory (when the version of ICU is 73). Therefore, if your PR changed any resource data files (ex: the ones contained within`icudata.jar`), you also need to update the corresponding resources files in order for your test to pass in the Maven build.
+
+#### Run a single test
```

Comment:
Nitpick: for completeness you might also add how to run a method:
``
mvn test -Dtest="ULocaleTest#TestGetAvailableByType" -DfailIfNoTests=false
```

There are test classes with many test methods, that can save some time for some people.